backend/app/api/v1/endpoints/auth.py

```
@@ -1,50 +1,61 @@
     result = AuthService().login(body)
-    response = ResponseHandler().success_response(data=result["user"], message="Login successful")
+    response = ResponseHandler().success_response(
+        data=result["user"], message="Login successful"
+    )
     set_auth_cookie(response, result["access_token"])
     return response
 
+
 @router.post("/logout")
 @auth_required()
 async def logout(request: Request):
```

Ruff (0.14.13)

Unused function argument: `request`

(ARG001)